server/neptune/workflows/internal/activities/terraform_test.go

```diff
@@ -11,6 +11,14 @@ import (
 	"go.temporal.io/sdk/testsuite"
 )
 
+type testStreamHandler struct{}
+
+func (t *testStreamHandler) Stream(ctx context.Context, jobID string, ch <-chan terraform.Line) error {
+	return nil
+}
+
+func (t *testStreamHandler) Close(ctx context.Context, jobID string) {}
+
```

Comment:
Caught this after merging but this isn't really testing anything? Ideally you should be asserting that these functions were called with the correct parameters.  I'm now fixing it in my PR.